resources/g2/sprites.json

```diff
@@ -2175,5 +2175,5049 @@
         "y_offset": -1,
         "palette": "keep",
         "forceBmp": true
+    },
```

Comment:
Please always keep the font sprites at the end. Not doing so risks running into problems later.